### PR TITLE
Format code

### PR DESCRIPTION
**Pull request by tenhobi**
Mainly, the code now uses file-scoped namespaces. Also, I removed a hangfire config from the docs. I also did autoformat for each file.

// I suggest hiding whitespace before viewing changes

### CODE REVIEW

File: src/HonzaBotner.Discord.Services/Commands/VoiceCommands.cs [hunk 1]
```diff
@@ -9,141 +9,140 @@
...
+    [Command("add")]
+    [Aliases("new")]
+    [Description("Create new voice channel. Users has 30 seconds to join.")]
+    [Priority(2)]
+    public async Task AddVoiceChannelWithLimitAndPublic(
+        CommandContext ctx,
+        [Description("Name of the channel.")] string name,
+        [Description("Limit number of members who can join. Use 0 to set to unlimited.")]
+        int limit = 0,
+        [Description("Set voice channel public.")]
```
Comment (stepech) [inline src/HonzaBotner.Discord.Services/Commands/VoiceCommands.cs:40]: The formating changes here and on line 68 of the same file contradict each other

**Review by stepech — APPROVED** [review]
I would maybe recommend finishing some developer formating configuration where we put in stone where do we put space and where we don't to avoid some of these issues in the future.

Approved based on a promise that @tenhobi will fix conflicts with #176 coming from this PR.

File: src/HonzaBotner.Discord.Services/Commands/VoiceCommands.cs (hunk 1 above)
Comment (tenhobi) [reply src/HonzaBotner.Discord.Services/Commands/VoiceCommands.cs:40]: We have the .editorconfig file that should set up everything. Not sure why the format tool chooses different styles, especially with annotations.

**Comment by stepech** [comment]
> Approved based on a promise that @tenhobi will fix conflicts with #176 coming from this PR.

That promise was broken quickly... #296

**Pull request merged**